server/test/sc/server/plugins/TestMove.kt

```diff
@@ -0,0 +1,13 @@
+package sc.server.plugins
+
+import sc.api.plugins.IMove
+import sc.server.helpers.TestTeam
+
+class TestMove(private val value: Int) : IMove {
+    fun perform(state: TestGameState) {
+        state.state = value
+        state.turn = state.turn + 1
```

Comment:
`++`?

Reply:
6c019052

Reply:
actually, this was already resolved in #339 but I forgot to update the branch first